src/App.tsx

```diff
@@ -15,48 +17,133 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type Good = string;
+type Sort = SortType | '';
+
+enum SortType {
+  SORT_FIELD_ALPHABET = 'alphabet',
+  SORT_FIELD_LENGTH = 'length',
+  SORT_FIELD_REVERSE = 'reverse',
+}
```

Comment:
It isn't the best idea to combine the sort field and reverse values in one enum, as for me. Consider using this enum only for sort fields and creating boolean variable for reverse